apps/studio/pages/api/ai/sql/supabase-mcp.ts

```diff
@@ -23,13 +23,14 @@ export async function createSupabaseMCPClient({
       accessToken,
       apiUrl,
     },
+    projectId,
```

Comment:
This option was missing (I think we just forgot to pass it on). Without it the MCP server would be scoped to the account-level.